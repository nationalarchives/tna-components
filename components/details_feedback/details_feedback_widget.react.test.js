import React from 'react';
import DetailsFeedbackWidget from './details_feedback_widget.react';
import TestRenderer from 'react-test-renderer';

// // Snapshot 1
it('renders correctly', () => {
  const tree = TestRenderer.create(<DetailsFeedbackWidget />).toJSON();
  expect(tree).toMatchSnapshot();
});
