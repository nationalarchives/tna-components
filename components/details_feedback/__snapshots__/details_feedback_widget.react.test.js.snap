// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <h2>
    Feedback
  </h2>,
  <div>
    <form
      onSubmit={[Function]}
    >
      <fieldset
        className="initial-question"
        id={null}
      >
        <legend>
          Could this page be improved?
        </legend>
        <button
          onClick={[Function]}
          type="button"
        >
          No
        </button>
        <button
          onClick={[Function]}
          type="button"
        >
          Yes
        </button>
      </fieldset>
    </form>
  </div>,
]
`;
